test/StandardSqlFormatterTest.js

```diff
@@ -202,6 +202,67 @@ describe("StandardSqlFormatter", function() {
         );
     });
 
+    it("formats simple SELECT", function() {
+        const result = sqlFormatter.format("SELECT 'value'", );
+        expect(result).toBe(
+            "SELECT\n" +
+            "  'value'\n"
+        );
+    });
+
+    it("formats simple SELECT", function() {
```

Comment:
It has a duplicate test description. Could we be more specific about why is this test different from the above one?